packages/runtime/src/useCases/transport/relationshipTemplates/CreateOwnRelationshipTemplate.ts

```diff
@@ -68,7 +68,16 @@ export class CreateOwnRelationshipTemplateUseCase extends UseCase<CreateOwnRelat
 
     private async validateRelationshipTemplateContent(content: any) {
         const transformedContent = Serializable.fromUnknown(content);
-        if (!(transformedContent instanceof RelationshipTemplateContent)) return;
+
+        if (!(transformedContent instanceof RelationshipTemplateContent || transformedContent instanceof ArbitraryRelationshipTemplateContent)) {
+            return RuntimeErrors.general.invalidPropertyValue(
+                "The content of a RelationshipTemplate must be of type RelationshipTemplateContent or ArbitraryRelationshipTemplateContent."
+            );
+        }
+
+        if (transformedContent instanceof ArbitraryRelationshipTemplateContent) {
+            return;
+        }
```

Comment:
```suggestion
        if (!(transformedContent instanceof RelationshipTemplateContent)) return;
```
this is future proof and will not explode if we add other types